drdroid_debug_toolkit/core/protos/playbooks/playbook.proto

```
@@ -49,6 +49,7 @@ import "core/protos/playbooks/source_task_definitions/mcp_task.proto";
 import "core/protos/playbooks/source_task_definitions/metabase_task.proto";
 import "core/protos/playbooks/source_task_definitions/confluence_cloud_task.proto";
 import "core/protos/playbooks/source_task_definitions/checkly_task.proto";
+import "core/protos/playbooks/source_task_definitions/site24x7_task.proto";
```

Buf (1.72.0)

[error] imported file does not exist

(COMPILE)